docs/contributing/coding-guidelines/ros-nodes/parameters.md

```diff
@@ -1,5 +1,79 @@
 # Parameters
 
-!!! warning
+Autoware has the following types of parameters:
 
-    Under Construction
+- `Node Parameter`: the reference parameter for the node package
+  - e.g., [the parameter for the `behavior_path_planner` package](https://github.com/autowarefoundation/autoware.universe/tree/main/planning/behavior_path_planner/config)
+- `Component Parameter`: the reference parameter for the component launch
+  - e.g., [the parameter for the `tier4_planning_launch` package](https://github.com/autowarefoundation/autoware.universe/tree/main/launch/tier4_planning_launch/config/)
+- `Launch Parameter`: [the reference parameter for the `autoware_launch` package](https://github.com/autowarefoundation/autoware_launch/tree/main/autoware_launch/config)
+
+`Launch Parameter` contains the same parameters as `Component Parameter`.
+`Component Parameter` contains all the parameters of `Launch Parameter`, and also parameters of multiple node instances from a single node package with different parameters.
+
+![parameter-architecture](images/parameter-architecture.svg){: style="width:900px"}
+
+## Parameters to be used
+
+### When launching autoware via the `autoware_launch` package
+
+In [`autoware.launch.xml`](https://github.com/autowarefoundation/autoware_launch/blob/main/autoware_launch/launch/autoware.launch.xml) of the `autoware_launch` package, parameters in the `autoware_launch` package (`Launch Parameter`) are used.
+For example, the `tier4_planning_launch` package receives `$(find-pkg-share autoware_launch)/config/tier4_planning_launch` as the `tier4_planning_launch_param_path` argument as follows.
+
+```xml
+<include file="$(find-pkg-share tier4_planning_launch)/launch/planning.launch.xml">
+  <arg name="tier4_planning_launch_param_path" value="$(find-pkg-share autoware_launch)/config/tier4_planning_launch"/>
+</include>
+```
+
+### When launching autoware via the component launch
```

Comment:
```suggestion
### When launching an individual component via the `tier4_COMPONENT_launch`
```
How about this?

Reply:
https://github.com/autowarefoundation/autoware-documentation/pull/171/commits/9ebdfe40092fcbd08b2741dccbc7dfce0a57afff
DONE. I remained the name component launch.